web/src/user_topic_popover.ts

```diff
@@ -54,11 +64,11 @@ export function initialize() {
                 const start_time = Date.now();
 
                 const visibility_policy = Number.parseInt(
-                    $(e.currentTarget).attr("data-visibility-policy"),
+                    $(e.currentTarget).attr("data-visibility-policy") ?? "0",
```

Comment:
We can null assert (`!`) this. Defaulting this to "0" is a little over defensive.